cohort_4/playbook/ACTIVITIES.md

```diff
@@ -1,6 +1,7 @@
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-generate-toc again -->
 **Table of Contents**
 
+- [Creating a safe space](#creating-a-safe-space)
```

Comment:
Perhaps it'd be more appropriate to put this near the bottom near `Using Slack` and `Communicating with the Group`? Should we also link this in the playbook's README with `Using Slack` and `Communicating with the Group`?